### ISSUE TITLE
Sampling-event datasets

### ISSUE THREAD

**Issue by CecSve** [gbif/doc-freshwater-data-publishing-guide#25]
Feedback for: https://docs.gbif-uat.org/freshwater-data-publishing-guide/en/#sampling-event-datasets 

I propose rephrasing the following section:

> When occurrence data have additional metadata describing sampling methods, this is called an event and the additional metadata are provided as sampling-event data ([Table 6](https://docs.gbif-uat.org/freshwater-data-publishing-guide/en/#table-06)).

to 

> Sampling-event data are structured and systematic surveys (i.e. periodical surveys or environmental monitoring) that must include metadata describing sampling methods ([Table 6](https://docs.gbif-uat.org/freshwater-data-publishing-guide/en/#table-06)).

for clarity.

The link in this sentence is dead (microhabitat):

> In addition, the field [samplingEffort](https://rs.tdwg.org/dwc/terms/samplingEffort) can be used to record the total effort spent on the event, for example, when there were multiple nets, multiple [microhabitat](https://docs.gbif-uat.org/freshwater-data-publishing-guide/en/#microhabitats) sampled, or multiple periods of time over which sampling occurred.

Please consider linking to relevant extension for sampling-event data, like the [Humboldt extension](https://rs.gbif.org/extension/eco/Humboldt_2024-04-16.xml) or [extended measurements or facts](https://rs.gbif.org/extension/obis/extended_measurement_or_fact_2023-08-28.xml)

**Comment by jenlento** [comment]
Thank you for the suggestions. 

We have adjusted the text about sampling events as suggested, with a small modification. The text reads, "i.e. periodical or singular surveys,  routine or one-time environmental monitoring" because it's important to note that some sampling event data may originate from single sampling events (i.e., often there is not funding for periodic sampling, but data from a single event can be provided).

We have fixed the link for microhabitat.

**Comment by CecSve** [comment]
In [table 6](https://docs.gbif-uat.org/freshwater-data-publishing-guide/en/#table-06) and [table 5](https://docs.gbif-uat.org/freshwater-data-publishing-guide/en/#table-05), eventDate appears as required. Strictly speaking, the recorded date could be shared in various ways, see: https://techdocs.gbif.org/en/data-processing/temporal-interpretation#occurrence-recorded-date. 

Perhaps the information from the 'advice for publishers' would be better to share?

> Advice for publishers
> 
> It is perfectly acceptable to provide only the eventDate field, and thus allow GBIF’s interpretation to fill in year, month, day, startDayOfYear and endDayOfYear. Alternatively, providing only year, year+month, or year+month+day (depending on available data) is also fine.
> 
> Avoid providing incorrect or incomplete data — do not set the day (as part of the eventDate or as the day) to 0 or 1 if only the year or month is known.

Although I am not sure how to capture it in a table format. I realise this recommendation is based on the conflicting information provided on the dataset classes page vs tech docs.

**Comment by jenlento** [comment]
@CecSve I'm also not sure about the best way to approach this. If we set it to recommended, I think we run the risk of people not including the date. But I understand that there are options to submit this information under different terms. What about something like, "Required, though year, month, day, or other terms could be used instead"?

**Comment by CecSve** [comment]
> [@CecSve](https://github.com/CecSve) I'm also not sure about the best way to approach this. If we set it to recommended, I think we run the risk of people not including the date. But I understand that there are options to submit this information under different terms. What about something like, "Required, though year, month, day, or other terms could be used instead"?

Yes, that seems like a good approach. And perhaps link to the technical documentation on the subject as a hyperlink of 'other terms could be used instead'?

**Comment by jenlento** [comment]
Updated as suggested.
